packages/frameworks/src/frameworks.ts

```diff
@@ -1618,6 +1618,58 @@ export const frameworks = [
     buildCommand: 'vite build',
     getOutputDirName: async () => 'dist',
   },
+  {
+    name: 'Astro',
+    slug: 'astro',
+    demo: 'https://astro.examples.vercel.com',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/astro.svg',
+    tagline:
+      'Astro is a next-generation static site generator that mixes lightning-fast performance with modern developer experience.',
+    description: 'An Astro site, using the default starter kit.',
+    website: 'https://astro.build',
+    envPrefix: 'SNOWPACK_PUBLIC_',
+    detectors: {
+      every: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"astro":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `astro build`',
+      },
+      devCommand: {
+        value: 'astro dev',
```

Comment:
Does `astro dev` read the `$PORT` env var already? If not, then it will need to be passed in a flag such as `astro dev --port $PORT`.

Reply:
> Does astro dev read the $PORT env var already?
It does not. I have filed an issue to discuss adding one

Would this need to be added to both `settings.devCommand.value` and `devCommand` properties?

Reply:
Yes